pkg/controller/test-images-distributor/test_images_distributor.go

```diff
@@ -430,7 +431,10 @@ func testInputImageStreamTagFilterFactory(l *logrus.Entry, ca agents.ConfigAgent
 			l.WithField("name", nn.String()).WithError(err).Error("Failed to get imagestreamname for imagestreamtag")
 			return false
 		}
-		if additionalImageStrams.Has(imageStreamName.String()) {
+		if additionalImageStreams.Has(imageStreamName.String()) {
+			return true
+		}
+		if additionalImageStreamNamespaces.Has(nn.Namespace) {
```

Comment:
We can move this above to line 20, the imageStream namespace and imagestreamTag namespace is guaranteed to be equal so we don't have to do this cheap check after using the indexer

Reply:
`line 20`: 420?

Reply:
err yeah, thats what I meant